code/index.html

```diff
@@ -2,21 +2,107 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" href="./news.png">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <nav> 
+        <img class="logo" src="./news.png" alt="logo"/>
+        <h1 class="site-name">The Generic News Site</h1>
+        <div class ="navigation" id="navigation-id">
+          <ul>
+            <li><a href="#big-news-id">Big News</a></li>
+            <li><a href="#small-news-id">Small News</a></li>
+            <li><a href="mailto: contact@genericnews.no.real.mail" target="_blank">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
+    <section class="big-news" id="big-news-id">
       <!-- The big news content -->
+      <img class="big-news-picture" src="https://picsum.photos/400/400" alt="big news"/>
```

Comment:
Maybe try to describe each image more in the `alt="big news"`? If the image somehow is broken, then we can see the description instead.